syncano/models/manager.py

```diff
@@ -828,6 +828,26 @@ def _get_endpoint_properties(self):
         return self.model._meta.resolve_endpoint(self.endpoint, defaults), defaults
 
 
+class RuntimeChoices(object):
```

Comment:
The list was in first version. Thought that would be nicer in such 'container'; the usage:

```
RUNTIMES = Script.please.get_runtimes()
# and later
Script.please.create(runtime_name=RUNTIMES.PHP, ...)

# rather than:

runtimes = Script.please.get_runtimes()
# and later
Script.please.create(runtime_name=runtimes[2], ...)

```